### ISSUE TITLE
Transpile option in Nuxt 2 includes whole bundle rather then an imported chunk

### ISSUE THREAD

**Issue by AndrewBogdanovTSS** [nuxt/nuxt.js#3622]
### Version

[v1.4.0](https://github.com/nuxt.js/releases/tag/v1.4.0)

### Reproduction link

[https://github.com/Atinux/nuxt-bt-test](https://github.com/Atinux/nuxt-bt-test)

### Steps to reproduce

Nuxt version: 2.0.0-25550715.56db988
Clone the repo
run `npm run build -a` to generate analyze report

### What is expected ?

Generated build should only include imported modules: Button and Modal in this specific case

### What is actually happening?

Build has all 153 modules of bt-vue included http://take.ms/fHtlg

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c7480">#c7480</a>)</em></sub></div>

**Comment by clarkdo** [comment]
It because `webpack` didn't tree shake the dead-code, I think `bootstrap-vue` should add `sideEffects` in its `package.json` for [Marking the file as side-effect-free](https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free)

You can also try to add `"sideEffects": false,` in `node_modules/bootstrap-vue/package.json` as a test, then run analyzer again.

Or you can import components as needed:

```
import Button from 'bootstrap-vue/es/components/button';
import Modal from 'bootstrap-vue/es/components/modal';
```

**Comment by AndrewBogdanovTSS** [comment]
Yeah, now it's working as it should. Thanks a lot for help, @clarkdo . Does it mean that a bug should be logged in bt-vue repo?  Does this "sideEffects" property have any side effects when added? :)
